### PR TITLE
chore: remove legacy link support

### PR DESCRIPTION
**Pull request by Chartman123**
Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

File: lib/Constants.php [hunk 1]
```diff
@@ -68,16 +68,13 @@ class Constants {
 	public const FORM_ACCESS_NOPUBLICSHARE = 0;
 	public const FORM_ACCESS_PERMITALLUSERS = 1;
 	public const FORM_ACCESS_SHOWTOALLUSERS = 2;
+	// still needed for Migrations
```
Comment (susnux) [inline lib/Constants.php:71]:
```suggestion
	/** @deprecated 5.0.0 still needed for Migrations
```

File: lib/Migration/Version050000Date20241005173955.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 */
+	public function postSchemaChange(IOutput $output, Closure $schemaClosure, array $options): void {
+		$qbUpdate = $this->db->getQueryBuilder();
+
+		$qbUpdate->update('forms_v2_forms')
```
Comment (susnux) [inline lib/Migration/Version050000Date20241005173955.php:33]: you also need to execute this then

Comment (Chartman123) [reply lib/Migration/Version050000Date20241005173955.php:33]: Good point 🙈😂

**Comment by Chartman123** [comment]
@susnx all comments should be fixed now, thanks for the review :)

**Review by susnux — APPROVED** [review]
looks good and a quick tests also worked :)

**Pull request merged**